storage-model/src/test/java/no/unit/nva/publication/storage/model/daos/DaoTest.java

```diff
@@ -11,72 +10,55 @@
 import static no.unit.nva.publication.storage.model.DatabaseConstants.PRIMARY_KEY_SORT_KEY_NAME;
 import static no.unit.nva.publication.storage.model.DatabaseConstants.RESOURCES_TABLE_NAME;
 import static no.unit.nva.publication.storage.model.DatabaseConstants.STATUS_INDEX_FIELD_PREFIX;
+import static no.unit.nva.publication.storage.model.daos.DaoUtils.parseAttributeValuesMap;
+import static no.unit.nva.publication.storage.model.daos.DaoUtils.toPutItemRequest;
 import static nva.commons.core.JsonUtils.objectMapper;
-import static nva.commons.core.attempt.Try.attempt;
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.core.Is.is;
 import static org.hamcrest.core.IsEqual.equalTo;
 import static org.hamcrest.core.IsNot.not;
 import static org.hamcrest.core.IsNull.nullValue;
 import static org.hamcrest.text.IsEmptyString.emptyString;
-import com.amazonaws.services.dynamodbv2.document.Item;
-import com.amazonaws.services.dynamodbv2.document.ItemUtils;
-import com.amazonaws.services.dynamodbv2.model.AttributeValue;
 import com.amazonaws.services.dynamodbv2.model.GetItemRequest;
 import com.amazonaws.services.dynamodbv2.model.GetItemResult;
-import com.amazonaws.services.dynamodbv2.model.PutItemRequest;
 import com.amazonaws.services.dynamodbv2.model.QueryRequest;
 import com.amazonaws.services.dynamodbv2.model.QueryResult;
 import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.databind.JsonNode;
 import java.net.MalformedURLException;
-import java.time.Clock;
-import java.util.Map;
 import java.util.stream.Stream;
 import no.unit.nva.model.exceptions.InvalidIssnException;
 import no.unit.nva.publication.service.ResourcesDynamoDbLocalTest;
-import no.unit.nva.publication.storage.model.DoiRequest;
-import no.unit.nva.publication.storage.model.Resource;
-import no.unit.nva.publication.storage.model.ResourceTest;
-import no.unit.nva.publication.storage.model.RowLevelSecurity;
-import no.unit.nva.publication.storage.model.WithIdentifier;
 import no.unit.nva.publication.storage.model.WithStatus;
 import nva.commons.core.SingletonCollector;
-import nva.commons.core.attempt.Try;
-import org.javers.core.Javers;
-import org.javers.core.JaversBuilder;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.MethodSource;
 
-class DaoTest extends ResourcesDynamoDbLocalTest {
-
-    public static final String EMPTY_VALUE_ERROR = "ValueMap was either null or empty";
-    public static ResourceTest resourceGenerator = new ResourceTest();
-    Javers javers = JaversBuilder.javers().build();
+public class DaoTest extends ResourcesDynamoDbLocalTest {
 
     @BeforeEach
     public void init() {
         super.init();
     }
 
-    @ParameterizedTest(name = "get Type Returns Name Of The Contained Object: {0}")
+    @ParameterizedTest(name = "getTypeReturnsNameOfTheContainedObject: {0}")
```

Comment:
Were these changes intentional?

Reply:
Yes. I thought to have use the same name as the test function. Maybe not the best idea.